### PR TITLE
Feature/whitelist direct access

### PR DESCRIPTION
**Pull request by TheFruxz**
Hey!

This is my second submission of a PaperMC/Paper (API) change.

In this proposal you can directly return players who are on the whitelist at the WhitelistToggleEvent.
I hope this change can help developers to handle the whitelist during a process.

With kind regards
Fruxz (@TheFruxz)

### CODE REVIEW

File: Spigot-API-Patches/0231-Maded-Title-objects-directly-sendable.patch [hunk 1]
```diff
@@ -0,0 +1,72 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fruxz <cedricspitzer@outlook.de>
+Date: Thu, 8 Oct 2020 11:06:46 +0200
+Subject: [PATCH] Maded Title-objects directly sendable
```
Comment (Proximyst) [inline Spigot-API-Patches/0231-Maded-Title-objects-directly-sendable.patch:4]: This is not within the scope of this PR.

File: Spigot-API-Patches/0232-WhitelistToggleEvent-can-now-return-whitelisted-play.patch [hunk 2]
```diff
@@ -0,0 +1,40 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fruxz <cedricspitzer@outlook.de>
+Date: Thu, 8 Oct 2020 11:41:32 +0200
+Subject: [PATCH] WhitelistToggleEvent can now return whitelisted players
```
Comment (Proximyst) [inline Spigot-API-Patches/0232-WhitelistToggleEvent-can-now-return-whitelisted-play.patch:4]: Fixup this patch into the patch which creates the class.

File: Spigot-API-Patches/0232-WhitelistToggleEvent-can-now-return-whitelisted-play.patch [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+@@ -27,6 +31,15 @@ public class WhitelistToggleEvent extends Event {
+         return enabled;
+     }
+ 
++    /**
++     * Gets the currently whitelisted players from the server-whitelist
++     *
++     * @return Currently whitelisted players from the server-whitelist
++     */
++    public Set<OfflinePlayer> getWhitelistedPlayers() {
```
Comment (Proximyst) [inline Spigot-API-Patches/0232-WhitelistToggleEvent-can-now-return-whitelisted-play.patch:34]:
```suggestion
+    @NotNull
+    public Set<OfflinePlayer> getWhitelistedPlayers() {
```

**Review by Proximyst — CHANGES_REQUESTED** [review]

**Comment by electronicboy** [comment]
1) you need to use branches to prevent PRs being mangled with other PRs
2) This patch also offers no real advantages to the API, all it does is further increase the potential maintenance cost for no real benefit, you also broke the build by not adding the proper nullability annotations, which would have shown up locally on compiling

Thanks for the PR, however

**Pull request closed without merging**